packages/web/app/src/lib/hooks/use-toggle.ts

```diff
@@ -1,10 +1,10 @@
 import { useState, useCallback } from 'react';
 
-export function useToggle(defaultValue: boolean) {
+export function useToggle(defaultValue = false): [boolean, () => void] {
```

Comment:
```suggestion
export function useToggle(defaultValue = false) {
```

Reply:
does it really matter? In this case, I explicitly provided a return type for the hook to avoid mistakes in function body

Reply:
just less manual typing

Reply:
I usually don't define the return type and let the typescript do the work